src/middlewared/middlewared/pytest/unit/plugins/enclosure/test_m_series_nvme.py

```diff
@@ -42,7 +42,7 @@ def test__m_series_nvme_enclosures(fs):
             )
         )
 
-        EnclosureService(middleware).m_series_nvme_enclosures()\
+        EnclosureService(middleware).map_nvme()\
```

Comment:
It was a typo
```suggestion
        EnclosureService(middleware).map_nvme()
```

Reply:
ahh, okay. I saw it but thought maybe something specific for pytest or something. Thanks!